config/routes.rb

```diff
@@ -75,7 +75,10 @@
     resources :fundraisers, only: :update
     resources :surveys, only: :update
     resources :texts, only: :update
-    resources :call_tools, only: :update
+    resources :call_tools, only: :update do
+      delete :delete_sound_clip, on: :member
```

Comment:
The controller action might still be called `delete_sound_clip` to make it clearer.